cmd/config/aws/create_role_cmd.go

```diff
@@ -0,0 +1,76 @@
+package aws
+
+import (
+	"fmt"
+	"github.com/armory/armory-cli/pkg/auth"
+	"github.com/armory/armory-cli/pkg/cmdUtils"
+	cliconfig "github.com/armory/armory-cli/pkg/config"
+	"github.com/armory/armory-cli/pkg/util"
+	"github.com/manifoldco/promptui"
+	"github.com/pkg/browser"
+	"github.com/spf13/cobra"
+	log "go.uber.org/zap"
+	"io"
+)
+
+const (
+	configCreateRoleShort = "AWS Create Role via CloudFormation Stack"
+	configCreateRoleLong  = "Use AWS CloudFormation quick create to run a template that allows Armory to assume a role to manage deployments\n\n" +
+		"For usage documentation, visit TODO"
+	configCreateRoleExample = "armory config aws create-role"
+	templateUrl             = "https://us-east-1.console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/quickcreate?templateURL=https://armory-cdaas-cloudformation.s3.us-west-2.amazonaws.com/iam-role-cfn.template&stackName=Armory-CDAAS-Role-Stack&param_AccountId=%s&param_ExternalId=%s"
+)
+
+func NewCreateRoleCmd(configuration *cliconfig.Configuration) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "create-role",
+		Aliases: []string{"create-role"},
+		Short:   configCreateRoleShort,
+		Long:    configCreateRoleLong,
+		Example: configCreateRoleExample,
+		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+			cmdUtils.ExecuteParentHooks(cmd, args)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return createRole(cmd, configuration)
+		},
+	}
+	return cmd
+}
+
+func createRole(cmd *cobra.Command, cli *cliconfig.Configuration) error {
+	orgID, err := cli.GetAuth().GetOrganizationId()
+	if err != nil {
+		return auth.ErrNotLoggedIn
+	}
+
+	envID, err := cli.GetAuth().GetEnvironmentId()
```

Comment:
this error is ignored, though it's unlikely that the first one would be nil and the second one wouldn't